code/game/objects/effects/countdown.dm

```diff
@@ -108,7 +108,7 @@
 	if(!istype(G))
 		return
 	else if(G.obj_integrity && !G.purpose_fulfilled)
-		return "<div align='center' valign='middle' style='position:relative; top:0px; left:6px'>[G.get_arrival_text(FALSE)]</div>"
+		return "<div align='center' valign='middle' style='position:relative; top:0px; left:6px'>[G.get_arrival_time(FALSE)]</div>"
```

Comment:
have you checked for any other places that might possibly need fixing?

Reply:
Nope, I tested this in-game, with the examine AND countdown

Reply:
and also with the initial countdown, grace period, and final countdown.

Reply:
so if you haven't checked, why not?

Reply:
cuz i tested all the situations in game

Reply:
that doesn't seem like valid reasoning. perhaps you should make sure you didn't _miss_ anything.

Reply:
I didn't. I TESTED EVERY POSSIBLE FUCKING SITUATION FOR FUCK'S SAKE.

Reply:
I'm not asking if you tested it. I'm asking if you checked for in-code uses of it that you haven't updated.

Reply:
No